src/aig-common/modules/generic/common-generic.module.ts

```diff
@@ -47,22 +43,20 @@ import { MatInputModule } from '@angular/material/input';
         FormsModule,
         ReactiveFormsModule,
 
-        AigGenericClientModule,
-
-        FlexLayoutModule,
         RouterModule,
+        AigGenericClientModule,
 
         MatTableModule,
         MatProgressSpinnerModule,
+        
         MatButtonModule,
         MatIconModule,
-        MatInputModule,
         MatFormFieldModule,
-        MatAutocompleteModule,
         MatSelectModule,
         MatDatepickerModule,
+        MatAutocompleteModule,
 
-
+        FlexLayoutModule,
     ],
     providers: [
         EopooTypeResolver,
```

Comment:
restore original of this file

Reply:
this is the original: 
[https://github.com/EneaSys/aig_console/blob/develop/src/aig-common/modules/generic/common-generic.module.ts](url)